lambda_functions/automate_s2_nbar_rolling_archive/execute_s2nbar/s3_sync.py

```diff
@@ -0,0 +1,181 @@
+import glob
```

Comment:
Is this file new code that you've written, or is it from somewhere else originally?